lib/poper/rule/fifty_char_summary.rb

```diff
@@ -1,10 +1,22 @@
 module Poper
   module Rule
     class FiftyCharSummary < Rule
-      MSG = 'Git commit message summary (first line) should be 50 chars or less'
-
       def check(message)
-        MSG if message.lines.first.chomp.length > 50
+        error_message if message.lines.first.chomp.length > character_limit
+      end
+
+      def enabled?
+        @config.summary_character_limit_enabled.to_s == 'true'
+      end
+
+      private
+
+      def character_limit
+        @config.summary_character_limit_number.to_i
+      end
+
+      def error_message
```

Comment:
I had to do that for the character count rules in order to interpolate the number of characters. The rest I changed for consistency.